### DISCUSSION TITLE
How to get started?

### DISCUSSION THREAD

**Discussion by Summersisle** [Opinionated-Blueprints/10-Books-Full-of-Rails#32] [Q&A] [answered]
I have followed the guide in the Wiki but I am still running into "No stops with this name exist". Is it possible to get a sample blueprint of a minimum viable set up?
![image](https://github.com/user-attachments/assets/9ffff536-45fa-4e7e-9de3-9f914fb8637b)

**Comment by Bocian-1** [comment]
The train is looking for a [coal] [provider chest] - [item parameter] station, but as the train says, it doesn't exist. You named your provider and requester stations with [virtual signal 1] instead of [coal]

**Reply by Summersisle** [reply]
But the station gets auto named that based off the input from the Parameterised build window. If I leave the group blank & the item coal the station is named [Parameter 0] [provider chest] - [item parameter]

Does the station name need to be changed manually?

**Reply by Bocian-1** [reply, marked as answer]
What you should've done is set the train's group to [virtual signal 1], matching the station's group instead of [coal] matching the resource

**Reply by Summersisle** [reply]
That works thank you!

**Comment by Rangelkent** [comment]
What guide in the wiki did you follow? I cant get this to work
